### ISSUE TITLE
Errors in the Qt3D modules with PySide2 5.12.4+ and Python 2

### ISSUE THREAD

**Issue by goanpeca** [spyder-ide/qtpy#195]
From https://github.com/spyder-ide/qtpy/pull/194#issuecomment-508862786

>It looks like the python 2.7 tests fail due to an update to shiboken2 as PySide has been bumped from 5.12.3 -> 5.13 on June 27.

>Previous run on master: https://circleci.com/gh/spyder-ide/qtpy/686
Latest run of master: https://circleci.com/gh/spyder-ide/qtpy/692

>Release history of Shiboken2: https://pypi.org/project/shiboken2/#history

>You'll need to address these changes at some point as anyone trying to use Qtpy on that version will experience an error when using the Qt3DAnimation module.

**Comment by ccordoba12** [comment]
@jschueller, could you help us with this one? Thanks!

**Comment by jschueller** [comment]
seems like a shiboken bug in python 2.x, it goes fine if I add ".copy()" inshibokensupport/signature/mapping.py:233
```
candidates = list(mod_name for mod_name in sys.modules.copy()
                          if self.module_valid(sys.modules[mod_name]))
```

**Comment by ccordoba12** [comment]
How do PySide2 releases work? Do you think this is going to be available in 5.13.1?

I say it so we can skip the test for the right versions.

**Comment by jschueller** [comment]
The change is against for the 5.13 release branch, if accepted it could be in the next.
Let's hear what they say first, I will keep you posted.

**Comment by ccordoba12** [comment]
Ok, thanks! We need to skip this so our test suite pass again. For now I'll do it for 5.13+, and when the fix is accepted we can change it accordingly.

**Comment by ccordoba12** [comment]
Sorry, the right skip was simply for Python 2.

**Comment by jschueller** [comment]
@ccordoba12 do you know if pytest uses threads to run these tests ?

**Comment by ccordoba12** [comment]
I think it runs them serially, unless you're using `pytest-xdist`.

**Comment by jschueller** [comment]
ok, thanks

**Comment by jschueller** [comment]
@ccordoba12 the fix got merged

**Comment by ccordoba12** [comment]
Great! Is it part of an actual release?

**Comment by jschueller** [comment]
Probably for 5.13.1

**Comment by ccordoba12** [comment]
Ok, please let me know the correct release the fix is going in, so we can fix things in our side.

**Comment by jschueller** [comment]
@ccordoba12 I confirm it got into v5.13.1
Looks like its going to be in 5.12.6 as well

**Comment by ccordoba12** [comment]
Ok, let's wait to see if it's included in 5.12.6
